package.json

```diff
@@ -143,6 +143,10 @@
     ]
   },
   "types": "lib/index.d.ts",
+  "resolutions": {
+    "@types/responselike": "1.0.0",
+    "got": "12.3.1"
+  },
```

Comment:
Relevant: https://classic.yarnpkg.com/lang/en/docs/selective-version-resolutions/

Surprised to see this in here. Did projen add it. Any risks?

Reply:
https://github.com/projen/projen/commit/b2b12bc60b45486e5aebe5b37dc5dba31f9b747d

> // Pin version of got and @types/responselike, see: https://github.com/sindresorhus/got/issues/2129

Risk seems low until it breaks something. `got` and `responseLike` are only used by projen. `(latest-version`/`package-json` deps)